posts/flat-file-exports/flat-files-compare.qmd

```diff
@@ -0,0 +1,440 @@
+ $ complexity                                                              : num  1.5 1.44 1.68 1.64 1.61 ...
+ $ errorCtEstimate                                                         : int  0 0 0 0 309 0
+ $ percOfDataFromError                                                     : num  0 0 0 0 0.547 ...
+ $ qcFlagsRNACounts                                                        : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ qcFlagsCellCounts                                                       : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ qcFlagsCellPropNeg                                                      : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ qcFlagsCellComplex                                                      : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ qcFlagsCellArea                                                         : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ qcCellsFlagged                                                          : logi  FALSE FALSE FALSE FALSE FALSE FALSE
+ $ median_negprobes                                                        : num  14 14 14 14 14 14
+ $ negprobes_quantile_0.9                                                  : num  25.1 25.1 25.1 25.1 25.1 25.1
+ $ median_RNA                                                              : int  51 51 51 51 51 51
+ $ RNA_quantile_0.9                                                        : num  166 166 166 166 166 166
+ $ nCell                                                                   : int  932 932 932 932 932 932
+ $ nCount                                                                  : int  614735 614735 614735 614735 614735 614735
+ $ nCountPerCell                                                           : num  660 660 660 660 660 ...
+ $ nFeaturePerCell                                                         : num  404 404 404 404 404 ...
+ $ propNegativeCellAvg                                                     : num  0.00058 0.00058 0.00058 0.00058 0.00058 ...
+ $ complexityCellAvg                                                       : num  1.54 1.54 1.54 1.54 1.54 ...
+ $ errorCtPerCellEstimate                                                  : num  100 100 100 100 100 ...
+ $ percOfDataFromErrorPerCell                                              : num  0.152 0.152 0.152 0.152 0.152 ...
+ $ qcFlagsFOV                                                              : chr  "Pass" "Pass" "Pass" "Pass" ...
+ $ i.median_negprobes                                                      : num  14 14 14 14 14 14
+ $ i.negprobes_quantile_0.9                                                : num  25.1 25.1 25.1 25.1 25.1 25.1
+ $ i.median_RNA                                                            : int  51 51 51 51 51 51
+ $ i.RNA_quantile_0.9                                                      : num  166 166 166 166 166 166
+```
+
+
+:::
+
+# Transcript coordinates file
```

Comment:
Can you add a sentence about the differences or note that this file has stayed largely the same?

Reply:
I have updated the text. 1a0416d378b30a2f45278643629781e3b497fd92